src/users/__tests__/userSelectors-test.js

```diff
@@ -208,3 +209,28 @@ describe('getUsersById', () => {
     expect(result).toEqual(expectedResult);
   });
 });
+
+describe('getOtherUsers', () => {
+  test('returns all users but current user', () => {
+    const state = deepFreeze({
+      users: [
+        { id: 1, email: 'abc@example.com', isActive: true },
+        { id: 2, email: 'def@example.com', isActive: true },
+        { id: 3, email: 'xyz@example.com', isActive: true },
```

Comment:
Use better emails to make it more obvious. For example `john@example.com`, `mark@example.com` and for the own email `me@example.com`.